app/src/main/java/me/ccrama/redditslide/Activities/LiveThread.java

```diff
@@ -203,62 +203,85 @@ protected Void doInBackground(Void... params) {
                         com.neovisionaries.ws.client.WebSocket ws = new WebSocketFactory().createSocket(thread.getWebsocketUrl());
                         ws.addListener(new WebSocketListener() {
                             @Override
-                            public void onStateChanged(com.neovisionaries.ws.client.WebSocket websocket, WebSocketState newState) throws Exception {
+                            public void onStateChanged(
```

Comment:
Yep. Reformat decided that none of these need to throw an exception... I didn't touch these directly but it IS accurate that they do not need that notation anymore...  *sigh*